### PR TITLE
Pageinfo deferredseq

### PR DESCRIPTION
**Pull request by raymondng**
The changes are pretty much all mechanical.

@leogrim @yingjieMiao @ymatsuda -- please take a quick look. Thanks.

### CODE REVIEW

File: kifi-backend/modules/shoebox/marketing [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 908d5800bd54c891b14da9fd2abc709433e7e22c
+Subproject commit a0fdc6f897368afbae5b4025c54e68665ee07cc3
```
Comment (raymondng) [inline kifi-backend/modules/shoebox/marketing:1]: Not intentional. Must be pulled when pulled from master.

**Comment by yingjieMiao** [comment]
@jaredpetker could help you on that submodule thing.

File: kifi-backend/modules/shoebox/app/com/keepit/model/PageInfoRepo.scala [hunk 2]
```diff
@@ -66,4 +66,9 @@ class PageInfoRepoImpl @Inject() (
       (for (f <- rows if f.uriId === uriId && f.state === PageInfoStates.ACTIVE) yield f).firstOption
     }
   }
+
+  override def assignSequenceNumbers(limit: Int = 20)(implicit session: RWSession): Int = {
+    assignSequenceNumbers(sequence, "page_info", limit)
+  }
+
```
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/model/PageInfoRepo.scala:73]: why we need this here? I thought this is in the trait.

**Comment by ymatsuda** [comment]
:+1:

File: kifi-backend/modules/shoebox/app/com/keepit/model/PageInfoRepo.scala (hunk 2 above)
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/model/PageInfoRepo.scala:73]: @yingjieMiao is right, you may want to pull the latest code, I made this logic generic and moved it up over the weekend.

File: kifi-backend/modules/shoebox/marketing (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/marketing:1]: Don't merge this one in. It'll bring the marketing site to an old commit.

Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/marketing:1]: @raymondng could checkout the specific commit in the marketing submodule.

**Comment by raymondng** [comment]
Thanks. No worries. Will submit another clean PR to avoid the marketing thing.

File: kifi-backend/modules/shoebox/app/com/keepit/model/PageInfoRepo.scala (hunk 2 above)
Comment (raymondng) [inline kifi-backend/modules/shoebox/app/com/keepit/model/PageInfoRepo.scala:73]: Yup just updated and noticed the change. Thanks.

**Pull request closed without merging**